### DISCUSSION TITLE
Does this work on Windows 7 32-Bit?

### DISCUSSION THREAD

**Discussion by Mythril382** [KadeArchive/Kade-Engine#646] [Q&A]

**Comment by isakube** [comment]
`lime build windows -32bit` i think

**Comment by Mythril382** [comment]
hmmm..... kay

**Comment by Mythril382** [comment]
wher can i find it?

**Comment by ACardboardBox1** [comment]
you put it in command prompt. also pretty sure this is a discussion

**Comment by Mythril382** [comment]
how? im kinda new

**Reply by isakube** [reply]
cd to the source code (terminal)

**Comment by Subsonic-Vaporeon** [comment]
Possibly, there might be a way of playing on windows 7
